playground.html

```diff
@@ -973,7 +929,13 @@
                 </header>
                 <p>Shapes are simple pictures made by drawing a geometric shape rather than loading an image file.</p>
```

Comment:
We might want to drop "simple" since we can do compound shapes now by nesting them in PathSets.